prow/jobs.md

```diff
@@ -73,6 +73,34 @@ that matches `trigger` will suffice. This is useful if you want to make one
 command that reruns all jobs. If unspecified, the default configuration makes
 `/test <job-name>` trigger the job.
 
+## Presets
+
+[`Presets`] can be used to define commonly reused values for a subset of fields
+for PodSpecs and BuildSpecs. The subset of fields chose was inspired by
+[PodPresets] which at time of writing are still in alpha. A preset config looks
+like:
+
+```yaml
+presets:
+- labels:                  # a job with these labels/values will have the preset applied
+    preset-foo-bar: "true" #   the covention we follow is preset-name: "true"
```

Comment:
might be worth noting that the label key must be unique

Reply:
done